### PR TITLE
LPS-169210 Add integration test for titleObjectField

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related to changes done in: https://issues.liferay.com/browse/LPS-169210

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java [hunk 1]
```diff
@@ -1302,6 +1333,26 @@ private void _assertSystemObjectFields(
...
+			String objectDefinitionName, String objectFieldName)
+		throws Exception {
+
+		SystemObjectDefinitionMetadata systemObjectDefinitionMetadata =
+			_systemObjectDefinitionMetadataRegistry.
+				getSystemObjectDefinitionMetadata(objectDefinitionName);
+
+		ObjectDefinition systemObjectDefinition =
+			_objectDefinitionLocalService.fetchSystemObjectDefinition(
+				systemObjectDefinitionMetadata.getName());
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java:1346]: @p-albuquerque do we really need to call the `getSystemObjectDefinitionMetadata`? Can't we `fetchSystemObjectDefinition` using the `objectDefinitionName`?

**Comment by p-albuquerque** [comment]
ci:stop

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Pull request closed without merging**